README.md

```diff
@@ -1,3 +1,16 @@
-# Ralf
+Ralf
+==============
 
-Ralf is a component of the Metaswitch Clearwater project, designed to act as the CTF in an IMS compliant deployment.
+Overview
+--------
+Ralf is a component of the Metaswitch Clearwater project, designed to act as the CTF (Charging Trigger Function) in an IMS compliant deployment. It converts JSON bodies in HTTP requests from IMS components into Diameter Rf ACRs. It uses memcached to store Rf session information for the duration of a session, and it uses [Chronos](https://github.com/Metaswitch/chronos) to send regular INTERIM ACRs to keep the session alive.
```

Comment:
Technically it's the CTF of the Clearwater nodes - other P-CSCFs or ASes will have/be their own CTFs.